dist/types.d.ts

```diff
@@ -7,9 +7,6 @@ interface PaystackCustomFields {
     variable_name: string;
     value: any;
 }
-interface PaystackMetadata {

```

Comment:
why did you remve it?

Reply:
The `PaystackMetadata` type isn't properly typed. I get type errors when I add arbitrary properties to the metadata object, which shouldn't be a problem.